### ISSUE TITLE
how to use mpv-stats script?

### ISSUE THREAD

**Issue by hbhslsr** [iina/iina#3081]
**System and IINA version:**

- macOS 10.15.6
- IINA 1.0.7

**Expected behavior:**
I try to load [mpv-stats](https://github.com/Argon-/mpv-stats) to see the statistics of the video. I have set the keymap using the method mentioned in #1053 but it still doesn't work. I don't know where to put the script **stats.lua**. the developer indicates that the script should be moved to `~/.config/mpv/scripts/` or `~/.mpv/scripts/folder`. what about IINA? Thank you for you attention. 

**Actual behavior:**

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**

**Comment by lhc70000** [comment]
You should set the mpv config directory in Preferences - Advanced so that mpv can load your scripts.

**Comment by varenc** [comment]
@hbhslsr that stats script is now built into MPV so you don't need to load an external script to get this to work.  Just change the keymap to call `script-binding display-stats-toggle` instead of `script-binding stats/display-stats-toggle`. 

For example, I've bound `i` so that it toggles the stats on and off.  (slightly different than MPV default)
![image](https://user-images.githubusercontent.com/167585/92682712-e6b0cb00-f2e5-11ea-8bc3-1aef01954d24.png) this just maps to an input conf of `i script-binding display-stats-toggle`


To show the different pages in the stats script, you have to setup other script-binding keymaps.  (key bindings made by lua scripts don't work in IINA).  Though oddly I can't get page #2, frame timings, to show up in IINA.  But the other pages work.  Here's my keymaps to show them.
![image](https://user-images.githubusercontent.com/167585/92683066-c2a1b980-f2e6-11ea-8316-97a52081e3e7.png)

**Comment by hooke007** [comment]
Yeah, I also strongly recommend to have a default key in iina to show the stats' information.

**Comment by vtns** [comment]
Somebody have any idea about why script-binding display-page-2 is not working?
